### ISSUE TITLE
reportMissingImports FIXED error: "Import "..extern.jaraco.text" could not be resolved"

### ISSUE THREAD

**Issue by drakeredwind01** [microsoft/pylance-release#2962]
quick question why does my linked quick nav not work here?
and should I fill in the rest of the information even though it is fixed?

- [Start](#Start)
- [TLDR fixed](#TLDR-fixed)

## Environment data

-   Language Server version: XXX
-   OS and version: XXX
-   Python version (& distribution if applicable, e.g. Anaconda): XXX

## Code Snippet

```python
XXX
```

## Repro Steps

1. XXX

## Expected behavior

XXX

## Actual behavior

XXX

## Logs

```
XXX
```

## Start
While filling this out I got the error
```
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "D:\documents\GitHub\studybud\env\lib\site-packages\django\urls\resolvers.py", line 604, in url_patterns
    iter(patterns)
TypeError: 'module' object is not iterable

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\drakeredwind01\Anaconda3\envs\firstconda\lib\threading.py", line 926, in _bootstrap_inner
    self.run()
  File "C:\Users\drakeredwind01\Anaconda3\envs\firstconda\lib\threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "D:\documents\GitHub\studybud\env\lib\site-packages\django\utils\autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "D:\documents\GitHub\studybud\env\lib\site-packages\django\core\management\commands\runserver.py", line 118, in inner_run
    self.check(display_num_errors=True)
  File "D:\documents\GitHub\studybud\env\lib\site-packages\django\core\management\base.py", line 423, in check
    databases=databases,
  File "D:\documents\GitHub\studybud\env\lib\site-packages\django\core\checks\registry.py", line 76, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "D:\documents\GitHub\studybud\env\lib\site-packages\django\core\checks\urls.py", line 40, in check_url_namespaces_unique
    all_namespaces = _load_all_namespaces(resolver)
  File "D:\documents\GitHub\studybud\env\lib\site-packages\django\core\checks\urls.py", line 67, in _load_all_namespaces
    namespaces.extend(_load_all_namespaces(pattern, current))
  File "D:\documents\GitHub\studybud\env\lib\site-packages\django\core\checks\urls.py", line 57, in _load_all_namespaces
    url_patterns = getattr(resolver, 'url_patterns', [])
  File "D:\documents\GitHub\studybud\env\lib\site-packages\django\utils\functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "D:\documents\GitHub\studybud\env\lib\site-packages\django\urls\resolvers.py", line 611, in url_patterns
    raise ImproperlyConfigured(msg.format(name=self.urlconf_name)) from e
django.core.exceptions.ImproperlyConfigured: The included URLconf '<module 'base.urls' from 'D:\\documents\\GitHub\\studybud\\base\\urls.py'>' does not appear to have any patterns in it. If you see valid patterns in the file then the issue is probably caused by a circular import.
```
So I googled `does not appear to have any patterns in it. If you see valid patterns in the file then the issue is probably caused by a circular import`
which took me to this specific [stack overflow ](https://stackoverflow.com/questions/34465954/trying-to-trace-a-circular-import-error-in-django)
where 
[dani-cs](https://stackoverflow.com/users/2877523/dani-cs)
[Share](https://stackoverflow.com/a/55349715/7619092)
with 56 upvotes wrote
```
for those who have the same error but still hasn't debugged their code, also check how you typed "urlpatterns"
having it mistyped or with dash/underscore will result to the same error
```
## After checking I confirmed that I had wrote
```
URLPatterns = [
    path('', views.home, name='home'),
    path('room/', views.room, name='room')
]
```
when it should have been 
```
urlpatterns = [
```
## TLDR-fixed
I wrote 
```
URLPatterns = [
```
when it should have been 
```
urlpatterns = [
```

**Comment by judej** [comment]
Closing, please reopen if there is a Pylance issue that you are still seeing.
